### PR TITLE
[WebUI] Fix 'Use All' Params not Respecting Hi-Res Fix

### PR DESCRIPTION
**Pull request by blhook**
This is a different source/base branch from https://github.com/invoke-ai/InvokeAI/pull/2823 but is otherwise the same content. `yarn build` was ran on this clean branch. 

## What was the problem/requirement? (What/Why)
As part of a [change in 2.3.0](https://github.com/invoke-ai/InvokeAI/commit/d74c4009cb671dd69768bfc0c6376a7ea74e358e), the high resolution fix was no longer being applied when 'Use all' was selected. This effectively meant that users had to manually analyze images to ensure that the parameters were set to match. 
~~Additionally, and never actually working, Upscaling and Face Restoration parameters were also not pulling through with the action, causing a similar usability issue.~~ See: https://github.com/invoke-ai/InvokeAI/pull/2823#issuecomment-1445530362

## What was the solution? (How)
This change adds a new reducer to the `postprocessingSlice` file, mimicking the `generationSlice` reducer to assign all parameters appropriate for the post processing options. This reducer assigns:
* Hi-res's toggle button only if the type is `txt2img`, since `img2img` hi-res was removed previously
* ~~Upscaling's toggle button, scale, denoising strength, and upscale strength~~
* ~~Face Restoration's toggle button, type, strength, and fidelity (if present/applicable)~~

### Minor
* Added `endOfLine: 'crlf'` to prettier's config to prevent all files from being checked out on Windows due to difference of line endings (and git not picking up those changes as modifications, causing ghost modified files from Git)

### Revision 2:
* Removed out upscaling and face restoration pulling of parameters
### Revision 3:
* More defensive coding for the `hires_fix` not present (assume false)

### Out of Scope
* Hi-res strength (applied as img2img strength in the initial image that is generated) is not in the metadata of the final image and can't be reconstructed easily
* Upscaling and face restoration have some peculiarities for multi-post processing outside of the UI, which complicates it enough to scope out of this PR. 

## How were these changes tested?
* `yarn dev` => Server started successfully
* Manual testing on the development server to ensure parameters pulled correctly
* `yarn build` => Success

## Notes
As with `generationSlice`, this code assumes `action.payload.image` is valid and doesn't do a formal check on it to ensure it is valid.

### CODE REVIEW

File: invokeai/frontend/.prettierrc.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 module.exports = {
   trailingComma: 'es5',
   tabWidth: 2,
+  endOfLine: 'crlf',
```
Comment (psychedelicious) [inline invokeai/frontend/.prettierrc.js:4]: Setting this to `crlf` results in prettier changing a bazillion line endings. Any reason to not use `auto`? Our eslint rule already uses `auto` so I think it should match.

File: invokeai/frontend/src/features/parameters/store/postprocessingSlice.ts [hunk 2]
```diff
@@ -40,6 +41,16 @@ export const postprocessingSlice = createSlice({
...
   reducers: {
+    setAllPostProcessingParameters: (
+      state,
+      action: PayloadAction<InvokeAI.Metadata>
+    ) => {
+      const { type, hires_fix } = action.payload.image;
+
+      if (type === 'txt2img' && typeof hires_fix === 'boolean')
+        state.hiresFix = hires_fix;
+      // Strength of img2img used in hires_fix is not currently exposed in the Metadata for the final image.
```
Comment (psychedelicious) [inline invokeai/frontend/src/features/parameters/store/postprocessingSlice.ts:52]: This doesn't handle the case where a image did not use hires, and hires is already enabled. Let's instead cast the value of `hires_fix` to bool to cover all situations:

```suggestion typescript
      if (type === 'txt2img') {
        // Strength of img2img used in hires_fix is not currently exposed in the Metadata for the final image.
        state.hiresFix = Boolean(hires_fix);
      }
```

**Review by psychedelicious — CHANGES_REQUESTED** [review]
Thanks for redoing this PR. I've requested two changes.

File: invokeai/frontend/src/features/parameters/store/postprocessingSlice.ts (hunk 2 above)
Comment (blhook) [reply invokeai/frontend/src/features/parameters/store/postprocessingSlice.ts:52]: > This doesn't handle the case where a image did not use hires 

The UI always has this field, even when false, but it seems the CLI doesn't handle this quite the same. Good catch -- will change. 

Other [similar assignments](https://github.com/invoke-ai/InvokeAI/blob/main/invokeai/frontend/src/features/parameters/store/generationSlice.ts#L290) may need to be changed in the future (outside of the scope here).

Comment (lstein) [reply invokeai/frontend/src/features/parameters/store/postprocessingSlice.ts]: I just saw this and added the (tiny bit of) code needed for the CLI to write `hires_fix` to the metadata. Let me know if more is needed.

**Comment by psychedelicious** [comment]
@blhook is this ready for review? Sorry if I missed a notification.

**Comment by blhook** [comment]
@psychedelicious , yes, it should be ready. I probably should have made a specific comment for it after the push (which was not too long after my replies to the conversation).

**Comment by psychedelicious** [comment]
Ah, good. I'll do my review this evening.

**Review by psychedelicious — APPROVED** [review]
All good, thank @blhook !

**Pull request merged**